pallets/committee/src/mock.rs

```diff
@@ -107,9 +116,27 @@ pub fn run_to_block(n: u64) {
 }
 
 // Build genesis storage according to the mock runtime.
-pub fn new_test_ext() -> sp_io::TestExternalities {
-    let t = frame_system::GenesisConfig::default()
+pub fn new_test_ext<I>(members: I) -> sp_io::TestExternalities
```

Comment:
```suggestion
pub fn new_test_ext(members: Vec<AccountId>) -> sp_io::TestExternalities
```
make this a vec instead?

Reply:
`IntoIterator<T>` includes `Vec` and the arguments could be shorter, for example

```rust
(PROPOSER_ACCOUNT_ID..PROPOSER_ACCOUNT_ID + 4)
// or
(
    PROPOSER_ACCOUNT_ID,
    PROPOSER_ACCOUNT_ID + 1,
    PROPOSER_ACCOUNT_ID + 2,
    PROPOSER_ACCOUNT_ID + 3,
)
```